### DISCUSSION TITLE
Add a new example for the whole ICoM domain

### DISCUSSION THREAD

**Discussion by changliao1025** [DOE-ICoM/liao-etal_2023_mosart_joh#6] [Ideas]
As suggested by @hydrotian and @tanzeli1982 , we will add an ICoM example to existing 4 watersheds. 
This example will include:

1. updated slope
2. updated width/depth function
3. mapping enabled
4. global ID, domain file added.

**Comment by tanzeli1982** [comment]
Thanks @changliao1025

**Comment by changliao1025** [comment]
These files are ready for testing. They are currently stored on Compy.
